### PR TITLE
introduce reports with jinja

### PR DESCRIPTION
**Pull request by 4383**
Generate analyze reports by using jinja2 to
merge usages between CLI output and web html outputs

### CODE REVIEW

**Comment by 4383** [comment]
I need to add changes on tests so wait for the moment

File: kojak/templates/cli.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% for module in analyze.modules -%}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Module: {{ module }}
+{{ module.imports|length }} {{ module.imports|length|pluralize("import")}}
```
Comment (hobbestigrou) [inline kojak/templates/cli.txt]: I think it's now possible to use the pluralize from jinja.

File: kojak/templates/cli.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+{{ module.classes|length }} {{ module.classes|length|pluralize("class", "classes")}}
```
Comment (hobbestigrou) [inline kojak/templates/cli.txt]: Same here.

File: kojak/templates/cli.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+{% include 'cli/imports.txt' %}
+{% include 'cli/classes.txt' %}
+{% endfor %}
+This project have:
+- {{ analyze.imports }} {{ analyze.imports|pluralize("import") }}
```
Comment (hobbestigrou) [inline kojak/templates/cli.txt]: Same here.

File: kojak/templates/cli.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+- {{ analyze.classes }} {{ analyze.classes|pluralize("class", "classes") }}
```
Comment (hobbestigrou) [inline kojak/templates/cli.txt]: Same here.

File: kojak/templates/cli/classes.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{% for cls in module.classes -%}
+    {% if loop.index == 1 -%}
+    Module {{ module.classes|length|pluralize("classe", "classes")}}:
```
Comment (hobbestigrou) [inline kojak/templates/cli/classes.txt]: Same here.

File: kojak/templates/cli/imports.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{% for import in module.imports -%}
+    {% if loop.index == 1 -%}
+    Module {{ module.imports|length|pluralize("import")}}:
```
Comment (hobbestigrou) [inline kojak/templates/cli/imports.txt]: Same here.

**Review by hobbestigrou — CHANGES_REQUESTED** [review]

**Comment by hobbestigrou** [comment]
Ok for me, I agree.

**Review by hobbestigrou — APPROVED** [review]

**Pull request merged**